translate_3to2.go

```diff
@@ -310,17 +310,20 @@ func translateNode3to2(n types.Node, fss []string) old.Node {
 
 func translateFiles3to2(files []types.File, fss []string) (ret []old.File) {
 	for _, f := range files {
-		// Overwrite defaults to true in spec 2 and false in spec 3
-		if f.Node.Overwrite == nil {
-			f.Node.Overwrite = boolPStrict(false)
```

Comment:
Also this code was previously mutating the spec 3 file in place instead of the new spec2 one.  If this were in Rust that would have never happened since we would have passed `&File` and not `&mut File` for sure.